packages/richtext-lexical/src/features/toolbars/fixed/client/Toolbar/index.tsx

```diff
@@ -103,6 +103,7 @@ function ToolbarGroupComponent({
           <ToolbarDropdown
             anchorElem={anchorElem}
             editor={editor}
+            group={group}
             groupKey={group.key}
```

Comment:
Could you remove group.key and group.items, since we're passing the entire group anyways?